CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [#219](https://github.com/nf-core/epitopeprediction/pull/219) - Fix `EXTERNAL_TOOLS_IMPORT`` container registry and bump version
 
+### `Removed`
+
+- [#221](https://github.com/nf-core/epitopeprediction/pull/221) - Depricate support of `GSvar` and variant `tsv` input files
```

Comment:
True, "remove" is better 👍🏼 

Regarding the `Removed` section, I would rather keep it, since that is also mentioned in the [doc](https://keepachangelog.com/en/1.0.0/) of keep a changelog (which is referred to in [CHANGELOG.md](https://github.com/nf-core/epitopeprediction/blob/master/CHANGELOG.md))